src/app/components/MediaLoader/index.tsx

```diff
@@ -145,6 +145,7 @@ type Props = {
   blocks: MediaBlock[];
   className?: string;
   embedded?: boolean;
+  episodePageIdentifier: string;
```

Comment:
We need to use a name which is more generic, because this media loader should not need to know about an episode page identifier. 

However, naming things is the hardest thing in the world. 

Given that it's the counterName that we're trying replace with this variable, I am thinking something like `counterNameOverride` but don't love that. Let's see if we can find something more suitable.

Reply:
I will have a think of a variable name!